### ISSUE TITLE
Terminology hard to follow

### ISSUE THREAD

**Issue by elear** [elear/mtgvenue#6]
From: AJS

There are some tricky bits of the definitions because
they seem gappy.  We have a requirement of the Venue under the
Facility division, a notion of "Venue environs" that is a little
tricky given the meaning of Venue (maybe that's Facility environs?
Something else?), and a "Venue contract" which is probably a Facility
or Hotel or something contract.  I don't have a lot more to say here
because I still want to be sure we're in agreement about the large
before I tackle the middlin'.  But the terminology is hard to follow,
which might be partly because of the two-dimensional categories.  I
_think_ another problem might be that we need to define separately
Location, maybe City?, and maybe something else -- Greater
Neighbourhood is all I can come up with -- to make this work
reasonably.  (This becomes obvious in 5.1 where the tension between
Venue and city is most stark.)

**Comment by eckelcu** [comment]
Eliot and Andrew to work offline on evaluation of definitions and return to list with proposal

**Comment by eckelcu** [comment]
No objections to wording in draft-07, therefore closing.
Reference https://www.ietf.org/mail-archive/web/mtgvenue/current/msg01053.html
